setup.cfg

```diff
@@ -1,7 +1,22 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+# E133: Closing bracket missing indentation
+# E203: whitespace before :, flake8 disagrees with PEP-8
+# E226: missing whitespace around arithmetic operator
+# E228: missing whitespace around bitwise or shift operator
+# N802: function name should be lowercase
+# N803: argument name should be lowercase
+# N806: variable in function should be lowercase
+# N812: lowercase imported as non lowercase
+# N813: camel case imported as lowercase
+# N815: mixed case imported as lowercase
+# N816: mixed case variable in class scope
+# W503: line break after binary operator, flake8 disagrees with PEP-8
+# W504: Line break occurred after a binary operator
+ignore = E133, E203, E226, E228, N802,
```

Comment:
I think E133 and E226  are impossible once you are enforcing black.

Reply:
Removed.